docs/contributor/contribute.rst

```diff
@@ -0,0 +1,6 @@
+Contributing to a lesson
```

Comment:
Contributions can be in form of questions i.e, not a part of the lesson (or so?). I'm not sure about the terminology here, @seanlip do you have any suggestions?

Reply:
I see your point... something more general like 'Contributing to Oppia' might be too vague. I'm wondering if we need to be _that_ technically accurate in the title if only one of the contributions doesn't directly pertain to a lesson?

Reply:
For my part I'm happy to leave this to @audstav's discretion. (See comment I posted above in this thread, as well.)